docs/versioned/flink/flink-getting-started.md

```diff
@@ -547,6 +547,33 @@ RewriteDataFilesActionResult result = Actions.forTable(table)
 
 For more doc about options of the rewrite files action, please see [RewriteDataFilesAction](../../../javadoc/{{% icebergVersion %}}/org/apache/iceberg/flink/actions/RewriteDataFilesAction.html)
 
+## Configuration.
+
+The following options can be used to tune the performance of the query execution.
+
+| Key                                             | Default | Type    | Description                                                                                                                      |
+|:-----------------------------------------------:| ------- | ------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| table.exec.iceberg.infer-source-parallelism     | True    | Boolean | If is false, parallelism of source are set by config.<br/>If is true, source parallelism is inferred according to splits number. |
+| table.exec.iceberg.infer-source-parallelism.max | 100     | Int     | Sets max infer parallelism for source operator.                                                                                  |
+| table.exec.iceberg.expose-split-locality-info   | None    | Boolean | Expose split host information to use Flink's locality aware split assigner.                                                      |
+
+Usage:
+
+```java
+// access flink configuration
+Configuration configuration = tEnv.getConfig().getConfiguration();
+// set low-level key-value options
+configuration.setStriFlinkConfigOptions.TABLE_EXEC_ICEBERG_INFER_SOURCE_PARALLELISMed", "true");
```

Comment:
Remove the extra `ed` from the `TABLE_EXEC_ICEBERG_INFER_SOURCE_PARALLELISMed` ?

Reply:
done